src/shared/components/palette/document/write-document-field.component.ts

```diff
@@ -198,9 +210,17 @@ export class WriteDocumentFieldComponent extends AbstractFieldWriteComponent imp
 
   getUploadedFileName() {
     if (this.uploadedDocument) {
-      return this.uploadedDocument.get('document_filename').value;
+      return this.uploadedDocument.get(WriteDocumentFieldDefinitios.DOCUMENT_FILENAME).value;
     } else {
       return undefined;
     }
   }
 }
+
+class WriteDocumentFieldDefinitios {
```

Comment:
typo here missing `n` towards the end --> WriteDocumentFieldDefinitions

Reply:
DONE